ui/src/notebooks/pipes/Data/FieldSelectors.tsx

```diff
@@ -16,7 +19,10 @@ const FieldSelectors: FC<Props> = ({fields}) => {
     (field: string): void => {
       let updated = field
       if (updated === selectedField) {
+        event('Deselecting Field in Flow Query Builder')
         updated = ''
+      } else {
+        event('Selecting Field in Flow Query Builder')
```

Comment:
might want to add which field was selected in here?

Reply:
Sounds good